pages/vote/elo.tsx

```diff
@@ -91,15 +91,32 @@ export default function Elo({ sessions }: EloProps): JSX.Element {
         onSessionChoice={sessionChoiceHandler}
       />
 
-      <StyledDrawButton
-        kind="link"
-        type="button"
-        onClick={() => {
-          sessionChoiceHandler(sessionPair.SubmissionA, sessionPair.SubmissionB, true)
-        }}
-      >
-        It's a Draw!
-      </StyledDrawButton>
+      <StyledEloButtonContainer>
+        <StyledVoteButton
+          kind="primary"
+          onClick={() => {
```

Comment:
As a bit of an unfortunate coincidence (mainly because it relied on some ... let's call them quirks ... of React), we've lost some useful functionality when you submitted your vote.

We would previously have removed focus from the buttons once they were submitted (to prevent accidental/deliberate submits of the next pair as well before they've had a chance to read it)

It would also have scrolled back to the top of the abstract when the new pair was presented.

Might be worth thinking about how we could get that functionality back, or if we can live without it

Reply:
Hmm, definitely should at least remove focus on the buttons. I'll see if I can recreate this behaviour explicitly.

The other option I was looking at was moving the button group into the `EloVote` component